### ISSUE TITLE
Add type annotation support.

### ISSUE THREAD

**Issue by trim21** [05bit/peewee-async#125]
A example:
```python
from typing import TypeVar, Type

T = TypeVar('T')

class Manager:
    async def get(self, source_: Type[T], *args, **kwargs) -> T:
        pass
```

**Comment by kalombos** [comment]
@trim21 thank you for your proposal i think we need mypy checker in CI if we add type annotation in the code

**Comment by kalombos** [comment]
Closed as duplicate of #224
